### PR TITLE
[Documentation] Fix the typo problems of NewToGluten

### PR DESCRIPTION
**Pull request by zuochunwei**
## What changes were proposed in this pull request?

(Please fill in changes proposed in this fix)

## How was this patch tested?

(Please explain how this patch was tested. E.g. unit tests, integration tests, manual tests)

(If this patch involves UI changes, please attach a screenshot; otherwise, remove this)

### CODE REVIEW

File: docs/developers/NewToGluten.md [hunk 1]
```diff
@@ -84,25 +83,24 @@ root@xx2:~ps ux | grep intellij
...
 
 ## Set up gluten project
 
-- Make sure you have compiled gluten
-- Load the gluten by File->Open, select <gluten_home/pom.xml>
-- Activate your profiles such as <backends-velox>, and Reload Maven Project, you will find all your need modules have been activated
-- Create breakpoint and debug as you wish, maybe you can try `CTRL+N` to find `TestOperator` to start your test
+- Make sure you have compiled gluten.
+- Load the gluten by File->Open, select <gluten_home/pom.xml>.
+- Activate your profiles such as <backends-velox>, and reload Maven Project, you will find all your need modules have been activated.
```
Comment (jinchengchenghh) [inline docs/developers/NewToGluten.md:96]: This `Reload` is the button description in IDEA

Comment (zuochunwei) [reply docs/developers/NewToGluten.md:96]: OK

File: docs/developers/NewToGluten.md [hunk 2]
```diff
@@ -325,14 +326,17 @@ jps
 ps ux | grep TestOperator
 ```
 
-Execute gdb command to debug
-
+Execute gdb command to debug:
 ```bash
 gdb attach <pid>
+or
```
Comment (jinchengchenghh) [inline docs/developers/NewToGluten.md:332]: # or

File: docs/developers/NewToGluten.md [hunk 3]
```diff
@@ -325,14 +326,17 @@ jps
...
+gdb -p <pid>
 ```
 
 ```bash
 gdb attach 1375551
+or
```
Comment (jinchengchenghh) [inline docs/developers/NewToGluten.md:338]: # or

Comment (zuochunwei) [reply docs/developers/NewToGluten.md:338]: ?

File: docs/developers/NewToGluten.md (hunk 2 above)
Comment (zuochunwei) [reply docs/developers/NewToGluten.md:332]: what?

Comment (jinchengchenghh) [reply docs/developers/NewToGluten.md]: `# or`

**Comment by zuochunwei** [comment]
@jinchengchenghh

**Review by jinchengchenghh — APPROVED** [review]

**Pull request merged**